src/app/services/constants.ts

```diff
@@ -81,6 +81,10 @@ export enum ACTION_TYPE {
     USER_DETAILS_UPDATE_RESPONSE_SUCCESS = "USER_DETAILS_UPDATE_RESPONSE_SUCCESS",
     USER_DETAILS_UPDATE_RESPONSE_FAILURE = "USER_DETAILS_UPDATE_RESPONSE_FAILURE",
 
+    USER_REGISTRATION_REQUEST = "USER_REGISTRATION",
```

Comment:
Small one, but just to keep this consistent with the rest of the constants in this file, shouldn't the value here be `USER_REGISTRATION_REQUEST`?

Reply:
It absolutely should be - updated.